image_provisioner/playbooks/roles/delete-oldest/tasks/main.yaml

```diff
@@ -0,0 +1,74 @@
+---
+# Abort if target_directory is not set.
+- name: Check for the target_directory variable
```

Comment:
I don't think we need this check since the role is run only when the target_directory variable is set.

Reply:
Since that check is in a different file I would like to keep the check here. If anyone tries to call this role without that variable set, they will know exactly what is wrong and what to fix.